scripts/py_matter_idl/matter_idl/generators/java/ClusterReadMapping.jinja

```diff
@@ -41,6 +41,19 @@ public class ClusterReadMapping {
           read{{cluster.name}}{{attribute.definition.name | upfirst}}CommandParams
         );
         result.put("read{{attribute.definition.name | upfirst}}Attribute", read{{cluster.name}}{{attribute.definition.name | upfirst}}AttributeInteractionInfo);
+      {% if attribute | isFabricScopedList(typeLookup) %}
+        InteractionInfo read{{cluster.name}}{{attribute.definition.name | upfirst}}AttributeWithFabricFilterInteractionInfo = new InteractionInfo(
+          (cluster, callback, commandArguments) -> {
+            ((ChipClusters.{{cluster.name}}Cluster) cluster).read{{attribute.definition.name | upfirst}}AttributeWithFabricFilter(
+              ({{ attribute | chipClustersCallbackName(typeLookup) }}) callback,
+              (Boolean) commandArguments.get("isFabricFiltered")
+            );
+          },
```

Comment:
It would be better for the class to have a "mUseFabricFilteredRead" that is true by default, and can be set true/false on the object. That field would be used in the argument to the read method, rather than needing to forward from the argument. That would avoid needing to have double the number of methods for fabric-scoped item, especially since Fabric-Filtered is the vast majority to default.

Reply:
We could have our methods call the right thing depending if isFabricFiltered exists in the map or not.

The ClusterReadMapping class should be immutable to not cause odd behaviors when different users interact.

Reply:
@tcarmelveilleux , @andy31415 : Thank you for your feedback and guidance. I would like to suggest that we want to remove this modification at this time. and We want to apply this PR. Then, we can check more compotable patch with new PR. please let me know your guidance.